content/blog/2019/mesh-federation/index.md

```diff
@@ -0,0 +1,1816 @@
+---
+title: Mesh Federation
+subtitle: Connect multiple service meshes ad hoc, with limited service exposure and strict cross-cluster access control
+description: Connect multiple service meshes ad hoc, with limited service exposure and strict cross-cluster access control.
+publishdate: 2019-08-12
+attribution: Vadim Eisenberg (IBM)
+keywords: [traffic-management,multicluster,security,gateway,tls]
+---
+
+Managing applications across multiple Kubernetes clusters is currently one of the hottest topics in Kubernetes and Istio
+communities. You may need to split your applications between multiple clusters for different reasons, for example:
+
+-  scalability limits of a single cluster
+-  separation of production/dev/test/staging environments
+-  different security and compliance requirements for different parts of the application
+-  running parts of the application on multiple public clouds, and on a private cloud
+
+Currently there are [three patterns](/docs/setup/kubernetes/install/multicluster/) in Istio to create a
+[multicluster service mesh](/docs/concepts/multicluster-deployments/#multicluster-service-mesh).
+A _multicluster service mesh_ is a single _logical_ service mesh, spread among multiple Kubernetes clusters.
+
+Such a service mesh has the following characteristics:
+
+- **uniform identical naming** of namespaces and services. The `withdraw` service in the `accounts` namespace in one
+cluster has the same functionality and API as the `withdraw` service in the `accounts` namespace in all other clusters
+in the mesh.
+- **expose-all by default**. All the services are exposed by default to all the clusters in the mesh.
+- **common identity and common trust** are established between all the clusters in the mesh.
```

Comment:
nit: "common identity and trust"?

Reply:
@nmittler Fixed in https://github.com/istio/istio.io/pull/4776/commits/fd0189f7fd67ab8f5997208d9e201756ff0e7718, thanks.